### DISCUSSION TITLE
[Feature Request]Provides an optional summary of remote system information

### DISCUSSION THREAD

**Discussion by pedoc** [kingToolbox/WindTerm#855] [Ideas]
## Idea
This article describes the idea of reading remote information from an ssh session and showing it, including but not limited to the following information (perhaps a configuration option can be provided to let the user decide)
1.CPU usage/cpu basic information
2. Memory usage / basic memory information
3. Process list
4. Disk information (especially the remaining disk space)
5. etc

Below is a screenshot of a similar functionality of a tool called finalshell(mainly for the red frame selection part):
![image](https://user-images.githubusercontent.com/6604230/176335059-57b484e9-a556-4e3c-8995-c95f6157d89c.png)

## Motivation
It is convenient for developers and operation and maintenance personnel to pay attention to the key indicators of the machine to determine whether it is working properly

**Comment by breezemit** [comment]
Version 2.6 supports some functions
but
1、remote system information graphics window 
2、 custom command parameter function, which can dynamically generate commands according to input parameters 
These two functions are very useful to meis very useful to me
